examples/fundamentals__typescript/cypress/integration/tests.spec.ts

```diff
@@ -0,0 +1,22 @@
+/* global window */
+/// <reference path="../types.d.ts" />
+
+before(() => {
+  // @ts-expect-error
+  window.add = (a, b) => a + b
```

Comment:
move it to fixtures test.html file :)

Reply:
It cannot be removed entirely because it shows that the type of Test Runner `window` fails because it's not extended.

I rewrote the test like below:

```js
  it('test extending AUTWindow', () => {
    // Test Runner window object doesn't have add() function.
    // So, it should fail the type check.
    // @ts-expect-error
    window.add = (a, b) => a + b

    cy.window().then((win) => {
      // AUT add() is defined in the fixture, test.html.
      // So, it should pass the type check.
      return win.add(2, 3)
    })
```